app/dashboard/src/utilities/appBaseUrl.ts

```diff
@@ -1,5 +1,5 @@
 /** @file The base URL of the app. */
-import * as appUtils from '#/appUtils'
+import { ALL_PATHS_REGEX } from '#/appUtils'
 
 /** The base URL of the app. */
-export const APP_BASE_URL = location.pathname.replace(appUtils.ALL_PATHS_REGEX, '')
+export const APP_BASE_URL = location.pathname.replace(ALL_PATHS_REGEX, '')
```

Comment:
Is stripping more than it should be or something so we have the wrong `APP_BASE_URL`?

Reply:
i don't think so, the app seems to be working fine apart from the wasm stuff